public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="./Turing-logo-small.png" />
```

Comment:
clean change, this is nice. May be best to place all of images inside a public folder, but this works